bitcoin/src/blockdata/transaction.rs

```diff
@@ -846,14 +852,21 @@ impl Transaction {
         Ok(())
     }
 
-    /// Is this a coin base transaction?
+    /// Checks if this is a coinbase transaction.
+    ///
+    /// The first transaction in the block distributes the mining reward and is called the coinbase
+    /// transaction. It is impossible to check if the transaction is first in the block, so this
+    /// function checks the structure of the transaction instead - the previous output must be
+    /// all-zeros (creates satoshis "out of thin air").
     pub fn is_coin_base(&self) -> bool {
         self.input.len() == 1 && self.input[0].previous_output.is_null()
     }
 
-    /// Returns `true` if the transaction itself opted in to be BIP-125-replaceable (RBF). This
-    /// **does not** cover the case where a transaction becomes replaceable due to ancestors being
-    /// RBF.
+    /// Returns `true` if the transaction itself opted in to be BIP-125-replaceable (RBF).
+    ///
+    /// This **does not** cover the case where a transaction becomes replaceable due to ancestors
+    /// being RBF. Please note that transactions may be replaced even if they do not include the RBF
+    /// signal: <https://bitcoinops.org/en/newsletters/2022/10/19/#transaction-replacement-option>.
```

Comment:
Yes, the wording is good. If you will touch this again maybe make it a bit more scary - add `## Warning` subtitle above the paragraph and add a bit of bold: " Please note that transactions **may be replaced** even if they **do not** include the RBF signal:"

And maybe to make it obvious, add: "**Incorrectly relying on RBF may lead to monetary loss!**"

(It may seem a bit too much but I noticed people miss serious security implications even in supposedly obvious contexts, so better stay on the safe side.)

Reply:
Agreed with all of these suggestions.

Reply:
Awesome, cheers. Done as suggested. Left it as a separate patch again since this is all docs improvements.